__tests__/data/blueshift_output.json

```diff
@@ -7,5 +7,423 @@
+  },
+  {
+    "body": {
+      "XML": {},
+      "FORM": {},
+      "JSON": {
+        "ip": "14.5.67.21",
+        "event": "identify",
+        "cookie": "1234abcd-efghijkj-1234kfjadslk-34iu123",
+        "os_name": "Android",
+        "latitude": "37.7672319",
+        "device_id": "3f034872-5e28-45a1-9eda-ce22a3e36d1a",
+        "longitude": "-122.4021353",
+        "event_uuid": "34abcd-efghijkj-1234kf",
+        "customer_id": "sampleusrRudder7",
+        "device_idfa": "3f034872-5e28-45a1-9eda-ce22a3e36d1a",
+        "device_idfv": "3f034872-5e28-45a1-9eda-ce22a3e36d1a",
+        "device_type": "Android",
+        "network_carrier": "Android",
+        "device_manufacturer": "Google"
+      },
+      "JSON_ARRAY": {}
+    },
+    "type": "REST",
+    "files": {},
+    "method": "POST",
+    "params": {},
+    "headers": {
+      "Content-Type": "application/json",
+      "Authorization": "Basic ZWZlYjRhMjlhYmE1ZTc1ZDk5YzhhMThhY2Q2MjBlYzE="
+    },
+    "version": "1",
+    "endpoint": "https://api.getblueshift.com/api/v1/event"
+  },
+  {
+    "body": {
+      "XML": {},
+      "FORM": {},
+      "JSON": {
+        "list_id": "35838",
+        "identifier_key": "customer_id",
+        "identifier_value": "rudderstack9"
+      },
+      "JSON_ARRAY": {}
+    },
+    "type": "REST",
+    "files": {},
+    "method": "PUT",
+    "params": {},
+    "headers": {
+      "Content-Type": "application/json",
+      "Authorization": "Basic YjRhMjlhYmE1ZTc1ZDk5YzhhMThhY2Q5MjBlYzFlMmU="
+    },
+    "version": "1",
+    "endpoint": "https://api.getblueshift.com/api/v1/custom_user_lists/add_user_to_list/35838"
+  },
+  {
+    "statusCode": 400,
+    "error": "[Blueshift]:: name and description are required to create an empty list, or List Id is required to add user.",
+    "statTags": {
+      "destination": "blueshift",
+      "stage": "transform",
+      "scope": "exception"
+    }
+  },
+  {
+    "statusCode": 400,
+    "error": "[Blueshift]:: name and description are required to create an empty list, or List Id is required to add user.",
+    "statTags": {
+      "destination": "blueshift",
+      "stage": "transform",
+      "scope": "exception"
+    }
+  },
+  {
+    "body": {
+      "XML": {},
+      "FORM": {},
+      "JSON": {
+        "ip": "14.5.67.21",
+        "event": "Product_purchased",
+        "cookie": "1234abcd-efghijkj-1234kfjadslk-34iu123",
+        "os_name": "Android",
+        "latitude": "37.7672319",
+        "device_id": "3f034872-5e28-45a1-9eda-ce22a3e36d1a",
+        "longitude": "-122.4021353",
+        "event_uuid": "34abcd-efghijkj-1234kf",
+        "customer_id": "sampleRudderstack9",
+        "device_idfa": "3f034872-5e28-45a1-9eda-ce22a3e36d1a",
+        "device_idfv": "3f034872-5e28-45a1-9eda-ce22a3e36d1a",
+        "device_type": "Android",
+        "network_carrier": "Android",
+        "device_manufacturer": "Google"
+      },
```

Comment:
Products purchased event should be having product info in properties which should be mapped also